### PR TITLE
Add stat constants in plugins

### PR DESCRIPTION
**Pull request by OciXCrom**
Replace all hardcoded numbers in stats plugins with the new constants from #509 + the HIT_ constants.

### CODE REVIEW

**Comment by Arkshine** [comment]
Please test/compile first before making a PR. It misses some constants includes for the {mod}stats plugins.

**Comment by OciXCrom** [comment]
@Arkshine Sorry, I didn't compile only one of the files and it turned out to have a problem. It's fixed now.

File: plugins/cstrike/statsx.sma [hunk 1]
```diff
@@ -752,26 +743,26 @@ format_rankstats(id, sBuffer[MAX_BUFFER_LENGTH + 1], iMyId = 0)
...
 	for (new i = 1; i < sizeof(L_BODY_PART); i++)
 	{
 		formatex(L_BODY_PART[i], charsmax(L_BODY_PART[]), "%L", id, BODY_PART[i])
 	}
 
 	iLen += formatex(sBuffer[iLen], charsmax(sBuffer) - iLen, "%10s:^n%10s: %d^n%10s: %d^n%10s: %d^n%10s: %d^n%10s: %d^n%10s: %d^n%10s: %d", "HITS",
-					L_BODY_PART[1], izBody[1], L_BODY_PART[2], izBody[2], L_BODY_PART[3], izBody[3], L_BODY_PART[4], izBody[4], L_BODY_PART[5],
-					izBody[5], L_BODY_PART[6], izBody[6], L_BODY_PART[7], izBody[7])
+					L_BODY_PART[HIT_HEAD], izBody[HIT_HEAD], L_BODY_PART[HIT_CHEST], izBody[HIT_CHEST], L_BODY_PART[HIT_STOMACH], izBody[HIT_STOMACH], L_BODY_PART[HIT_LEFTARM], izBody[HIT_LEFTARM], L_BODY_PART[HIT_RIGHTARM],
+					izBody[HIT_RIGHTARM], L_BODY_PART[HIT_RIGHTLEG], izBody[HIT_RIGHTLEG], L_BODY_PART[HIT_RIGHTLEG], izBody[HIT_RIGHTLEG])
```
Comment (Arkshine) [inline plugins/cstrike/statsx.sma]: `L_BODY_PART[6], izBody[6]` should be `L_BODY_PART[HIT_LEFTLEG], izBody[HIT_LEFTLEG]`. I thought you would have changed in an automatic way but looks like it's needed to review carefully each change.

Comment (OciXCrom) [reply plugins/cstrike/statsx.sma]: @Arkshine Fixed.

**Comment by Arkshine** [comment]
It was painful to review, but it looks good now. Read three times to make sure.

**Comment by WPMGPRoSToTeMa** [comment]
@Arkshine I think you could just check if it produces the same plugin binaries.

**Pull request merged**